packages/elements-core/src/components/Docs/Model/Model.tsx

```diff
@@ -24,49 +25,53 @@ const ModelComponent: React.FC<ModelProps> = ({ data: unresolvedData, className,
 
   const example = React.useMemo(() => generateExampleFromJsonSchema(data), [data]);
 
-  return (
-    <Box className={cn('Model', className)}>
-      {!layoutOptions?.noHeading && title !== undefined && (
+  const shouldDisplayHeader = !layoutOptions?.noHeading && title !== undefined;
+
+  const header = (shouldDisplayHeader || isInternal) && (
```

Comment:
Is it worth to create new components for 3 lines of code?

What would be inside such a component? 🤔

Reply:
I didn't mean a new component, just extracting those two parts and using separate conditions for displaying them, instead of this `||`. It's not critical though, I'm 100% fine with merging this

Reply:
Well, I had to do it this way in order for paddings to be always ok in the component.

Empty fragment `<></>` is unfortunately still considered a component in React, which causes lots of padding to appear, even if there is no header and no badges.

Awkward? Yes.

Will it work fine? Also yes. 😂